fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -1603,6 +1606,12 @@ public Map<Long, TableSnapshotInfo> collectBaseTableSnapshotInfos(MaterializedVi
                         throw new DmlException("Failed to copy lake table: %s", table.getName());
                     }
                     tables.put(table.getId(), new TableSnapshotInfo(baseTableInfo, copied));
+                } else if (table.isPaimonTable()) {
```

Comment:
Please don't add a special branch at here, move the connector specific code to `ConnectorPartitionTraits` instead

Reply:
ok